spec/specreview_checklist.md

```diff
@@ -0,0 +1,73 @@
+# Spec Review Checklist
+
+1. Spec PR
+  - [ ] PR uses [template](https://github.com/jakartaee/specifications/blob/master/pull_request_template.md)
+  - [ ] Directory of form {spec}/x.y
+  - [ ] PDF of form {spec}-spec-x.y.pdf ("-spec" preferred but not required)
+  - [ ] HTML of form {spec}-spec-x.y.html ("-spec" preferred but not required)
+  - [ ] Index page {spec}/x.y/_index.md following (TBD) [template](https://github.com/eclipse/microprofile/blob/master/spec/spec_page_template.md)
+  - [ ] Staging repository link of the form
```

Comment:
Staging repo format should be https://oss.sonatype.org/content/repositories/orgeclipsemicroprofile-xxx/, e.g. https://oss.sonatype.org/content/repositories/orgeclipsemicroprofile-1351/